src/pages/studyView/UserSelections.tsx

```diff
@@ -92,6 +93,32 @@ export default class UserSelections extends React.Component<IUserSelectionsProps
             return acc;
         }, components);
 
+        // All custom charts
+        if(!_.isEmpty(this.props.customChartsFilter)) {
+            _.reduce((this.props.customChartsFilter), (acc, content:string[], key:string) => {
+                const chartMeta = this.props.attributesMetaSet[key];
+                if (chartMeta) {
+                    acc.push(
+                        <div className={styles.parentGroupLogic}>
+                            <GroupLogic
+                                components={[
+                                    <span className={styles.filterClinicalAttrName}>{chartMeta.displayName}</span>,
+                                    <GroupLogic components={content.map(label => {
+                                        return <PillTag
+                                            content={label}
+                                            backgroundColor={STUDY_VIEW_CONFIG.colors.theme.clinicalFilterContent}
+                                            onDelete={() => this.props.updateCustomChartFilter(chartMeta, _.remove(content, value => value !== label))}
+                                        />
+                                    })} operation={'or'} group={false}/>
+                                ]}
+                                operation={':'}
+                                group={false}/></div>
```

Comment:
move the `</div>` to the next line, so that it aligns with the opening `<div>` tag